Revit_oM/Requests/FIlterByDBTypeName.cs

```diff
@@ -20,25 +20,19 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using BH.oM.Base;
 using BH.oM.Data.Requests;
-using System;
-using System.Collections.Generic;
 using System.ComponentModel;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace BH.oM.Adapters.Revit
+namespace BH.oM.Adapters.Revit.Requests
 {
-    [Description("IRequest that filters all elements of given Autodesk.Revit.DB type.")]
-    public class DBTypeNameRequest : IRequest
+    [Description("IRequest that filters all elements of given Autodesk.Revit.DB type. Information about types can be found in the Revit API documentation.")]
+    public class FilterByDBTypeName : IRequest
```

Comment:
The file name has a capital `I` in `Filter` by mistake, please rename the file appropriately :smile: